daemon/tests/harness/mod.rs

```diff
@@ -64,7 +63,8 @@ impl Maker {
         let (oracle, monitor, wallet) = mocks::create_actors(&mocks);
         mocks.mock_common_empty_handlers().await;
 
-        let wallet_addr = wallet.create(None).spawn_global();
+        let (wallet_addr, wallet_fut) = wallet.create(None).run();
+        tokio::spawn(wallet_fut);
```

Comment:
Not a blocker but it would be good to always strive for a minimal diff. This change doesn't add anything as far as I can tell?

Reply:
it's a preparation to spot the stray `tokio::spawn` calls - I will need to ban `spawn_global()` as well. 
I'll keep this change as it's minimal if you're ok with it

Reply:
We can disable the tokio extension for `xtra` then we don't have `spawn_global` available :)